massa-pos-exports/src/pos_final_state.rs

```diff
@@ -482,4 +614,107 @@ impl PoSFinalState {
             StreamingStep::Finished(None)
         }
     }
+
+    /// Used to set the initial rolls after bootstrapping
+    pub fn set_initial_rolls(&mut self, initial_rolls: BTreeMap<Address, u64>) {
```

Comment:
Why not make the feilds public, and remove the setters?

Reply:
Done in f67dd591f21f38de7095af9fccd1cb66c4c2fe1d (+ combined the fields in InitialState struct)